jscomp/core/js_pass_debug.ml

```diff
@@ -22,7 +22,7 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA. *)
 
-#if (defined BS_BROWSER || defined BS_RELEASE_BUILD)
```

Comment:
I am not sure why this was here, but I don't see why it can not be included in the playground build.

Reply:
You might have been reading it wrong. This was already included in the playground.

One version is for debugging.

Reply:
Sorry, I didn't express myself correctly. I meant why the version of `dump` that actually dumps code was not being included in the playground.